Earth Ability Implemented

Belize22 opened the pull request:
- The player can now climb walls when the earth ability is active.
- Various corner cases have been accounted for to ensure that climbing can only be done while the earth ability is active.
- Some player settings were modified in order to capture the most intuitive player control for the player.

Belize22 committed:
Earth Ability Implemented

- When earth ability is trigger, the player can now climb walls.
- Various tweaks have been made to ensure that climbing
is only prevalent while the earth ability is active.
- Added some previous properties in the level demo branch to maintain the optimal settings for player control.

Megan-VanHumbeck committed:
Merge remote-tracking branch 'origin/dev' into earth-ability

Megan-VanHumbeck reviewed (changes requested):
Both before and after fixing merge issues I was not able to get the player to climb walls. Could you please take another look at this to see what changed and broke the feature?

Belize22 committed:
Modified some settings

- The vertical colliders were given back the wall material.
- The climbing speed was tweaked to fit what the climbing speed of the level should be (within relative context of the demo level).

Belize22 committed:
Merge branch 'dev' into earth-ability

Belize22 committed:
Tweaked Merge conflict details.

- Mostly involving the update and fixed update functions.
